pkg/runtime/upgrade.go

```diff
@@ -17,23 +17,22 @@ package runtime
 import (
 	"fmt"
 	"path/filepath"
-	"strings"
 )
 
 const (
-	chmodCmd       = `chmod +x %s/*`
-	mvCmd          = `mv %s/* /usr/bin`
-	getNodeNameCmd = `$(uname -n | tr '[A-Z]' '[a-z]')`
-	drainCmd       = `kubectl drain ` + getNodeNameCmd + ` --ignore-daemonsets`
-	upgradeCmd     = `kubeadm upgrade %s`
-	restartCmd     = `systemctl daemon-reload && systemctl restart kubelet`
-	uncordonCmd    = `kubectl uncordon ` + getNodeNameCmd
+	chmodCmd          = `chmod +x %s/*`
+	mvCmd             = `mv %s/* /usr/bin`
+	getNodeNameCmd    = `$(uname -n | tr '[A-Z]' '[a-z]')`
+	drainCmd          = `kubectl drain ` + getNodeNameCmd + ` --ignore-daemonsets`
```

Comment:
All things like kubectl operation, could we use kubernetes open API to make it? 

We should avoid using binary executing as possible? 

This would make code clear and robust.

Just my confusion?